velox/serializers/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 add_library(velox_presto_serializer PrestoSerializer.cpp
                                     UnsafeRowSerializer.cpp)
 
-target_link_libraries(velox_presto_serializer velox_vector)
+target_link_libraries(velox_presto_serializer velox_dwio_common velox_vector)
```

Comment:
I'm not sure if it is a good idea to add this dependency. What do we need from dwio::common? Can we refactor to extract what we need?

Reply:
We need this file https://github.com/facebookincubator/velox/blob/main/velox/dwio/common/Common.h, I would suggest to rename this file to Compression.h, and extract it to velox_compression.